sdk/keyvault/keyvault-secrets/src/lro/recover/operation.ts

```diff
@@ -58,7 +58,7 @@ async function update(
       state.isCompleted = true;
     } catch (_) {}
     if (!state.isCompleted) {
-      state.result = await client.recoverDeletedSecret(name, { requestOptions });
+      state.result = await client.recoverDeletedSecret(name, requestOptions);
```

Comment:
`requestOptions` here is of the type `RequestOptionsBase`
`recoverDeletedSecret()` takes `OperationOptions` which has property `requestOptions` of type `RequestOptionsBase`

So, I believe this is not the change we want to make...

Reply:
On my way!

Reply:
Done!